aws/rust-runtime/aws-config/src/test_case.rs

```diff
@@ -126,12 +128,36 @@ impl TestEnvironment {
                 .with_fs(self.fs.clone())
                 .with_env(self.env.clone())
                 .with_connector(DynConnector::new(connector.clone()))
-                .with_sleep(InstantSleep)
+                .with_sleep(TokioSleep::new())
```

Comment:
Was this intentional?

Reply:
yeah, I think we end up with a poll somewhere now so instant sleep wins and times out the request